blog/2023-11-13-introducing-stately-sky/index.mdx

```diff
@@ -0,0 +1,72 @@
+
+![GIF demonstrating flow generation process](./generateFlow.gif)
+
+After that, developers can interact with their “live actor” in the same way they’ve been able to work with other XState machines. There is an SDK for Sky-based actors, but the syntax is virtually the same, save for the initial connection and configuration:
+
+```tsx
+const actor = actorFromStately(
+  {
+    url: 'url of the running actor',
+    sessionId: 'unique session id for multiplayer',
+  },
+  skyConfig, //provided after initializing sky in your local dev environment
+);
+```
+
+The snippet above is all that’s needed in code to connect an application to the live running actor in Stately Sky. For details on all the configuration steps, check out the “[Getting Started](https://stately.ai/docs/stately-sky-getting-started)” guide that walks through the full setup.
+
+### Multiplayer: batteries included
+
+Sky removes the complexity of spinning up a running web server in a cloud platform or hosting it on premise. Thanks to [PartyKit](https://docs.partykit.io/), real-time communication and multiplayer come built-in for the developer. Users don’t have to concern themselves with wrangling and scaling concurrent live connections in Sky. If a client needs to interact with a live actor, they need only the `sessionId` and the rest is handled for them. Both Sky and PartyKit are built using CloudFlare, allowing us to scale with the needs of the user.
+
+## What can you do with Sky?
+
+Since Sky actors are built using the same tools, the good news is that everything users can build in the editor can work with Sky The developer experience is exactly the same. After connecting to their live actors, users can inspect and send events just like they can with any other XState machine.
+
+### Use Cases
+
+There are several instances where users would benefit from a live actor, particularly one that can accept requests from multiple sources. Running state machines as actors in Sky gives developers the readability of statecharts with the durability and fault tolerance of cloud compute. For example, long-running workflows where time is a serious constraint can be easily implemented and maintained within Sky. Workflows like:
+
+- Multiplayer-collaboration sessions, like whiteboarding, document editing, and gaming.
+- Asynchronous workflows, like fetching large amounts of data in the background
+- Long-running backend processes like medical patient onboarding flows or inventory management
+
+### How does it work? (architecture overview)
+
+![Component diagram image](./componentDiagram.png)
```

Comment:
```suggestion
![Diagram showing Stately Sky at the top, which communicates with the Sky SDK inside the user app. Stately Sky also communicates with PartyKit’s state orchestator which in turn talks to the PartyKit Scheduler for delayed transitions and to “machineHub”, which is built on Cloudflare Workers for platforms for stateless compute.](./componentDiagram.png)
```

Reply:
wowser! I never thought to make these link annotations so verbose. Makes sense from an accessibility perspective for sure though.